src/app/page.js

```diff
@@ -1,95 +1,94 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+'use client'
+import { useRef } from "react";
+import Popup from 'reactjs-popup';
 
-export default function Home() {
-  return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <p>
-          Get started by editing&nbsp;
-          <code className={styles.code}>src/app/page.js</code>
-        </p>
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{" "}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
+export default function Login() {
+  const usernameRef = useRef();
+  const passwordRef = useRef();
+  const emailRef = useRef();
+
+  const blueButton = {
+    border:"solid",
+    borderRadius:"20px",
+    borderColor:"blue",
+    padding:"10px",
+    color:"white",
+    background:"blue",
+    display:"block",
+    marginRight:"auto",
+    marginLeft:"auto",
+    cursor:"pointer"
+  }
+
+  async function LoginUserWithDetails(){
```

Comment:
You need to 
- Setup a [proper router](https://reactrouter.com/en/main) for client side routing. 
- Store JWT token returned on successful Login, because you will need it for all requests